src/main/java/se/knowit/bookitevent/controller/EventController.java

```diff
@@ -0,0 +1,36 @@
+package se.knowit.bookitevent.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+import se.knowit.bookitevent.model.Event;
+import se.knowit.bookitevent.service.EventService;
+
+import java.util.Set;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/v1/")

```

Comment:
I prefer to put the ending slash on the mapping annotation of each method instead on the mapping of the class.

Psuedo example
```
@RequestMapping("/api/v1")
class {
    @GetMapping("/event/{id}")
    funtion getEvent()

    @PutMapping("/event")
}
```

I think this makes it clearer that it is a path that is specified for every mapping.